README.md

```diff
@@ -1,24 +1,28 @@
-Verify Service Provider
-=======================
+# Verify Service Provider
 
-Verify Service Provider is a Service that provides support for the communication
-between Verify Hub and a Relying Party.
+GOV.UK Verify uses SAML (Security Assertion Markup Language) to securely exchange information about identities. A Relying Party can use Verify Service Provider to handle all SAML communication to and from the Verify Hub.
 
-Please refer to the [techinical onboarding guide](https://alphagov.github.io/rp-onboarding-tech-docs/) to understand how services can technically integrate with Verify.
+Using Verify Service Provider will make it easier to:
+* connect multiple services with GOV.UK Verify - you only need one instance of Verify Service Provider
+* handle certificate rotations - you can host multiple certificates at a time
 
-Setup
------
+You will need to host Verify Service Provider on your own infrastructure.
+
+Using Verify Service Provider is just one part of connecting to GOV.UK Verify. Refer to the [technical onboarding guide](https://alphagov.github.io/rp-onboarding-tech-docs/) for more information about connecting to GOV.UK Verify.
+
+## Setup
 
 ### Download
 
-Release versions of the Verify Service Provider are available upon request from idasupport+onboarding@digital.cabinet-office.gov.uk
```

Comment:
Ah, this isn't true anymore, releases are available [here](https://github.com/alphagov/verify-service-provider/releases) - documentation was wrong before though.